src/header/HeaderButtons.js

```diff
@@ -5,12 +5,11 @@ import './header.css';
 import icon from './user-icon1.png';
 import { Link } from 'react-router-dom';
 import { Overlay, Popover, Button, Row } from 'react-bootstrap';
-
+import axios from 'axios'; 
 import './loginmenu.css';
 import { useState, useRef } from 'react';
 
 
-
 function Greeting(props) {
```

Comment:
Is it possible that file should contain one component and inside there are all internal methods and fields?

Reply:
@fazlmakhsudov , it woulbe correct, but it's not my component. I prefer don't change code that wasn't written by me if there is no use